wrangler-transform/src/main/java/io/cdap/wrangler/Wrangler.java

```diff
@@ -646,6 +717,11 @@ public static class Config extends PluginConfig {
     static final String NAME_SCHEMA = "schema";
     static final String NAME_ON_ERROR = "on-error";
 
+    @Name(NAME_SQL_EXECUTION)
+    @Description("Toggle to configure execution language between JEXL and SQL")
```

Comment:
JEXL is a technical term that many users won't understand. It's also not technically correct, as many directives are direct java implementations that don't use jexl scripting. Not sure what would be a better term though. Native? Default?